client/src/components/Overview.tsx

```diff
@@ -0,0 +1,359 @@
+import { ArrowDownIcon, ArrowUpIcon } from '@radix-ui/react-icons';
+import { DashboardData } from '../types/FileData';
+import { Box } from '../ui/Box';
+import { Flex } from '../ui/Flex';
+import { Heading } from '../ui/Heading';
+import { Text } from '../ui/Text';
+import {
+  getLastBlockTime,
+  getNextCycle,
+  microToStacks,
+  numberWithCommas,
+  numFormatter,
+} from '../utils';
+import { PriceEvolution } from './PriceEvolution';
+
+interface OverviewProps {
+  statsData: DashboardData;
+}
+
+export const Overview = ({ statsData }: OverviewProps) => {
+  return (
+    <Box
+      css={{
+        '@lg': {
+          flex: 1,
+        },
+      }}
+    >
+      <Heading css={{ mb: '$5', '@lg': { mb: '$10' } }} as="h1" size={'2xl'}>
+        Overview
+      </Heading>
+
+      <Box
+        css={{
+          backgroundColor: '$gray2',
+          mb: '$5',
+          gap: '$3',
+
+          '@lg': {
+            display: 'flex',
+            py: '$5',
+          },
+        }}
+      >
+        <Box
+          css={{
+            mb: '$6',
+            position: 'relative',
+            width: '100%',
+            height: 300,
+          }}
+        >
+          <Box
+            css={{
+              position: 'absolute',
+              width: '100%',
+              height: '100%',
+              left: 0,
+            }}
+          >
+            <Text as="h3" css={{ pl: '$4', fontSize: '$1', color: '$gray9' }}>
+              Price evolution (last 30 days)
+            </Text>
+            <PriceEvolution priceEvolution={statsData.stxPriceEvolution} />
+          </Box>
+        </Box>
+        <Box
+          css={{
+            display: 'none',
+            width: 2,
+            height: 250,
+            backgroundColor: '$gray6',
+            my: 'auto',
+
+            '@lg': {
+              display: 'block',
+            },
+          }}
+        />
+
+        <Flex css={{ p: '$3' }} direction={'column'} gap="5">
+          <Box>
+            <Text as="h3" css={{ mb: '$1', fontSize: '$1', color: '$gray9' }}>
+              STX price
+            </Text>
+            <Flex
+              css={{
+                gap: '$2',
+
+                '@lg': {
+                  gap: 0,
+                },
+              }}
+            >
+              <Heading size={'2xl'} css={{ color: '$orange11' }}>
+                <Box css={{ color: '$orange9' }} as="span">
+                  $
+                </Box>
+                {statsData.priceData?.blockstack.usd}
+              </Heading>
+              <Text
+                css={{
+                  fontSize: '$1',
+                  color:
+                    statsData.priceData.blockstack.usd_24h_change > 0
+                      ? '#23A77F'
```

Comment:
can we add these values to the theme?

Reply:
sure will add! Will speak to Quentin about adding some red values to the design system for negative percentages